hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/DropHoodieTableCommand.scala

```diff
@@ -69,35 +72,41 @@ case class DropHoodieTableCommand(
     assert(table.tableType != CatalogTableType.VIEW)
 
     val basePath = hoodieCatalogTable.tableLocation
+    val catalogProperties = hoodieCatalogTable.catalogProperties
+    val tableConfig = hoodieCatalogTable.tableConfig
```

Comment:
See the failed UTs, to call this should make sure the hudi table path exists (hoodieCatalogTable.hoodieTableExist is true).